docs/tutorials/exptime.rst

```diff
@@ -0,0 +1,492 @@
+    >>> plt.xlabel(ccd_response_wls.unit)
+
+.. plot::
+
+    import astropy.units as u
+
+    import warnings
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astroplan import FixedTarget
+    from astroquery.gaia import Gaia
+
+    hatp11 = FixedTarget.from_name('Hat-p-11')
+
+    # width / height of search:
+    width = u.Quantity(1, u.arcmin)
+    height = u.Quantity(1, u.arcmin)
+    search_results = Gaia.query_object_async(coordinate=hatp11.coord, width=width, height=height)
+
+    # the queried star should be the one nearest to the given coordinates
+    search_results.add_index('dist', unique=True)
+    hatp11_info = search_results.loc['dist', min(search_results['dist'])]
+
+    T_eff = round(hatp11_info['teff_val'], -2)  # round to nearest 100 K
+
+    stellar_radius = hatp11_info['radius_val'] * u.R_sun
+    parallax = hatp11_info['parallax'] * u.mas
+    # parallax is given by Gaia in milliarcseconds, so convert to parsec:
+    distance = parallax.to(u.parsec, equivalencies=u.parallax())
+
+    distance_scale = float(stellar_radius / distance) ** 2 / np.pi
+
+    from astropy.io import fits
+
+    flux_url = ('ftp://phoenix.astro.physik.uni-goettingen.de/v2.0/HiResFITS/'
+           'PHOENIX-ACES-AGSS-COND-2011/Z-0.0/lte{T_eff:05d}-{log_g:1.2f}-0.0.PHOENIX-'
+           'ACES-AGSS-COND-2011-HiRes.fits').format(T_eff=int(T_eff), log_g=4.5)
+    wavelength_url = ('ftp://phoenix.astro.physik.uni-goettingen.de/v2.0/HiResFITS/'
+                      'WAVE_PHOENIX-ACES-AGSS-COND-2011.fits')
+
+    flux_of_target = fits.getdata(flux_url) * distance_scale
+    waveset_of_target = fits.getdata(wavelength_url)
+
+    flux_of_target = flux_of_target * (u.erg / u.s / u.cm ** 2 / u.cm)
+    waveset_of_target = waveset_of_target * u.Angstrom
+
+    from astroplan import Observer
+
+    observer = Observer.at_site('apo')
+
+    from synphot.spectrum import SpectralElement
+    from astropy.utils.data import download_file
+    from astroplan import Telescope
+
+    diameter = 3.5 * u.m
+    gain = 1.9 * (u.ct / u.adu)
+
+    # model the bandpass:
+    svo_link = ('http://svo2.cab.inta-csic.es/' +
+               'theory/fps3/fps.php?ID=')
+    filt_path = download_file(svo_link + 'SLOAN/SDSS.rprime_filter')
+    bandpass = SpectralElement.from_file(filt_path)
+
+    telescope = Telescope(diameter, bandpass, gain, ccd_response='default')
+    ccd_response_wls, ccd_response = telescope.ccd_response
+    plt.plot(ccd_response_wls, ccd_response)
+    plt.xlabel(ccd_response_wls.unit)
+    plt.show()
+
+.. _exptime-getting_the_exposure_time:
+
+Getting the Exposure Time
+=========================
+
+Then specify the signal to noise ratio you wish to reach and
+call the :func:`~astroplan.exptime_from_ccd_snr()` function:
+
+.. code-block:: python
+
+    >>> from astroplan import exptime_from_ccd_snr
+
+    >>> snr = 100
+    >>> exptime_from_ccd_snr(snr, waveset_of_target, flux_of_target, observer, telescope)
+    0.0022556432s
+
```

Comment:
Could you please add a note here that says something to the effect of: "S/N is computed following equations X-Y from Howell (ZZZZ). See the docstring for more details".